src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,57 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+    private Object[] keys = new Object[MAX_SIZE];
+    private Object[] values = new Object[MAX_SIZE];
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```

0 is default value for int